### ISSUE TITLE
Storybook plugin deprecated

### ISSUE THREAD

**Issue by daniele-zurico** [Capgemini/dcx-react-library#413]
deprecating @storybook/addon-knobs in favor of @storybook/addon-controls

Documentation: https://www.npmjs.com/package/@storybook/addon-knobs

Please follow these instructions:
```
git checkout release/0.7
git pull
git checkout -b 'feature/storybook-deprecated-plugin'
```
create a PR with base branch `release/0.7`

**Comment by Jaya-Rathore** [comment]
1. As @storybook/addon-knobs has already been replaced in the code and no warnings encountered related to this plugin, no changes required.
2.  Deprecating @storybook/addon-docs/blocks in favor of  @storybook/addon-docs to resolve the warning while running storybook.
Documentation : https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-scoped-blocks-imports

**Comment by Jaya-Rathore** [comment]
1. No changes required for @storybook/addon-knobs in favor of @storybook/addon-controls as there are no warnings encountered .
2. Deprecating  @storybook/addon-docs/blocks in favor of  @storybook/addon-docs. 
Documentation: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-scoped-blocks-imports

**Comment by daniele-zurico** [comment]
closed by #426
